todo-app/db/migrate.ts

```diff
@@ -0,0 +1,14 @@
+import { migrate } from "drizzle-orm/postgres-js/migrator";
+import db from "./connection";
+
+const migrateDB = async () => {
+  console.log("migration started...");
+  await migrate(db, { migrationsFolder: "migrations" });
+  console.log("migration ended...");
+};
+
+migrateDB()
+  .then((data) => console.log(data))
+  .catch((error) => {
+    throw new Error(error);
```

Comment:
I mean this makes no sense. You catch an error, to throw it again?
This is like saving a child from drowning and the throwing it back into the ocean